build.sc

```diff
@@ -78,7 +82,10 @@ object main extends MillModule {
     }
   }
   object api extends MillApiModule{
+    //def moduleDeps = Seq(core)
     def ivyDeps = Agg(
+      ivy"ch.epfl.scala:bsp4j:2.0.0-M4",
```

Comment:
Why must `main.api` and `main.core` depend on BSP? If it's a separate module, then it should be a separate module and everyone else shouldn't have to depend on it